test/mocha/10-test.js

```diff
@@ -45,3 +45,72 @@ describe('bedrock-jsonld-document-loader', () => {
     });
   });
 });
+
+describe('httpClientHandler', () => {
+  it('properly gets document', async () => {
+    let result;
+    let error;
+    const documentUrl = 'https://httpbin.org/json';
+
+    try {
+      result = await httpClientHandler.get({url: documentUrl});
+    } catch(e) {
+      error = e;
+    }
+    assertNoError(error);
+    should.exist(result);
+    result.should.eql({
+      contextUrl: null,
+      document: {
+        slideshow: {
+          author: 'Yours Truly',
+          date: 'date of publication',
+          slides: [
+            {
+              title: 'Wake up to WonderWidgets!',
+              type: 'all'
+            },
+            {
+              items: [
+                'Why <em>WonderWidgets</em> are great',
+                'Who <em>buys</em> WonderWidgets'
+              ],
+              title: 'Overview',
+              type: 'all'
+            }
+          ],
+          title: 'Sample Slide Show'
+        }
+      },
+      documentUrl: 'https://httpbin.org/json'
+    });
+  });
+  it('throws error if url does not start with "http"', async () => {
+    let result;
+    let error;
+    const url = `urn:zcap:xyz`;
```

Comment:
Can use regular quotes:
```suggestion
    const url = 'urn:zcap:xyz';
```

Reply:
Fixed 3cc5711